test/transpile/bundle.spec.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from "path";
+import * as util from "../util";
+import { transpileProject } from "../../src";
+
+const projectDir = path.join(__dirname, "bundle");
+const inputProject = path.join(projectDir, "tsconfig.json");
+
+test("should transpile into one file", () => {
+    const transpileResult = transpileProject(inputProject);
+
+    expect(transpileResult.diagnostics).not.toHaveErrorDiagnostics();
+    expect(transpileResult.emitResult.length).toBe(1);
+
+    const { name, text } = transpileResult.emitResult[0];
+    // Verify the name is as specified in tsconfig
+    expect(path.join(name)).toBe(path.join(projectDir, "bundle.lua"));
```

Comment:
That `path.join(name)` looks a bit hacky, maybe instead we may have a helper in `run.ts`, like `buildVirtualProject`, but with a config file? It also would make paths relative, so we'll be able to simplify `basic.spec.ts` as well.

Reply:
That should not have been there, I removed it.